base/test_helpers_driver.go

```diff
@@ -0,0 +1,29 @@
+package base
+
+import (
+	"helm.sh/helm/v3/pkg/cli"
+
+	"k8s.io/apimachinery/pkg/runtime"
+	dynamicfake "k8s.io/client-go/dynamic/fake"
+)
+
+// initKubeFake initialize the Kube clientset with a fake
+func (kd *KubeDriver) initKubeFake(objects ...runtime.Object) {
```

Comment:
We should not use fake methods on real objects. Let us please make them functions instead. This is true even if the methods are not exposed externally.
***
If you look at the k8s APIs and Helm APIs, they both adhere to the above pattern.

Reply:
Turns out I have made the same error as above while developing the `driver` package. If you wish, you can fix this in `driver`, or I will do so in my upcoming PR.

Reply:
done including the `driver` package.